### PR TITLE
Add release notes for v0.2.0

### PR DESCRIPTION
**Pull request by flavio**
Add the release notes for v0.2.0

Fixes https://github.com/kubewarden/kubewarden-controller/issues/34

### CODE REVIEW

File: RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  mutating: true
+  settings:
+    allowed_capabilities:
+    - CHOWN
+    required_drop_capabilities:
+    - NET_ADMIN
+```
+
+Some considerations worth of note:
+  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` where **optional** string attributes. Now they are array of
```
Comment (ereslibre) [inline RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md:65]:
```suggestion
  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` were **optional** string attributes. Now they are array of
```

File: RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    strings that must have at least one element.
+  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` where **optional** string attributes.
```
Comment (ereslibre) [inline RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md:67]:
```suggestion
  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` were **optional** string attributes. 
```

File: RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` where **optional** string attributes. Now they are array of
+    strings that must have at least one element.
+  * With `v1alpha1`, `apiGroups`, `apiVersions`, `resources` where **optional** string attributes. 
+    When left empty they would default to the `"*"` value.
+
+This is reflected in the example above, as you can see the `v1alpha2` resource explicitly defines all these attributes.
+
+### Upgrade steps
+
+This is a list of steps to perform to upgrade to the `v0.2.0` release.
```
Comment (ereslibre) [inline RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md:74]:
```suggestion
This is a list of steps to perform the upgrade to the `v0.2.0` release.
```

File: RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+#### Install new Custom Resource Definitions
+
+To install the new CRD definition execute the following commnad:
+```console
+kubectl apply -f https://raw.githubusercontent.com/kubewarden/helm-charts/kubewarden-controller-0.1.12/charts/kubewarden-controller/crds/clusteradmissionpolicies.yaml
+````
+
+#### Upgrade your policies
+
+Upgrade all the currently definied `ClusterAdmissionPolicy` resources to use the `v1alpha2` version.
```
Comment (ereslibre) [inline RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md:98]:
```suggestion
Upgrade all the currently defined `ClusterAdmissionPolicy` resources to use the `v1alpha2` version.
```

File: RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+#### Upgrade the helm release
+
+It's time to upgrade the helm release, this can be done via:
+
+```console
+helm upgrade kubewarden-controller --version 0.1.12 kubewarden/kubewarden-controller
+```
+
+Once this is done, the new version of the `kubewarden-controller` will be running on our cluster. The `policy-server` instance will keep operating as expected during the whole time. It won't even be restarted, unless you didn't just migrate
+your policies from `v1alpha1` to `v1alpha2`, but you also changed some details about the `ClusterAdmissionPolicy` resource.
```
Comment (ereslibre) [inline RELEASE_NOTES/RELEASE_NOTES_v0.2.0.md:114]:
```suggestion
Once this is done, the new version of the `kubewarden-controller` will be running on our cluster. The `policy-server` instance will keep operating as expected during the whole time. It won't even be restarted, unless you  changed some details about the `ClusterAdmissionPolicy` resource apart from the version itself.
```

**Review by ereslibre — CHANGES_REQUESTED** [review]
Thank you @flavio, some comments

**Comment by flavio** [comment]
@ereslibre I've incorporated all your fixes, thanks!

**Pull request merged**